projects/angular/src/wizard/API.md

```diff
@@ -0,0 +1,42 @@
+# Wizard-clarity - Style API
+
+| CSS Variable Name          | Description                                                        |
+| -------------------------- | ------------------------------------------------------------------ |
+|  --clr-wizard-main-bgcolor | Changes the background color in wizard element.
+|  --clr-wizard-stepnav-bgcolor | Changes the background color of the side navigation bar for all wizard elements..
+|  --clr-wizard-main-text-color | Changes the color of text in form validation for successfully filling information; in alt net, changes the color of text for checkbox text and in step navigation changes the text color of required information to proceed.
```

Comment:
I am lost. What is "alt net"? 
Thanks for probably checking all the places it's applied, but I don't understand the description.
I think we could safely stay with the simple "Wizard main text color" here.

Reply:
Updated, As suggested